src/map/skill.cpp

```diff
@@ -2057,9 +2057,6 @@ int skill_additional_effect(struct block_list* src, struct block_list *bl, uint1
 	case RL_AM_BLAST:
 		sc_start(src,bl,SC_ANTI_M_BLAST,20 + 10 * skill_lv,skill_lv,skill_get_time2(skill_id,skill_lv));
 		break;
-	case RL_HAMMER_OF_GOD:
-		sc_start(src,bl,SC_STUN,100,skill_lv,skill_get_time2(skill_id,skill_lv));
-		break;
```

Comment:
Hammer of God needs a bit of a revamp. It no longer targets a random area (whether or not the target is marked); it is now always centered around the target. The skill was simplified to work like a normal area of effect skill. The implementation is a copy of RL_D_TAIL, except that it sends a clif_skill_poseffect packet instead.